assets/css/style.css

```diff
@@ -161,3 +161,27 @@ p {
 		width: 60%;
 	}
 }
+.card{
```

Comment:
Add some fixed with to fit all content (make cards smaller and changes other params to accompany that)